charts/cloud-query/Chart.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: v2
+name: cloud-query
```

Comment:
It would honestly be easier to define this entirely within the console chart, but if you want it to be a separate chart, you'll need to add the same versioning logic we do for the `controller` chart in gh actions, since it'll be released with the same tagging all at once.